src/main/java/coresearch/cvurl/io/exception/BadFileException.java

```diff
@@ -0,0 +1,8 @@
+package coresearch.cvurl.io.exception;
+
+public class BadFileException extends RuntimeException {
```

Comment:
An exception name is poor. It doesn't represent what happened. Smth like MultipartFileFormException suits better

Reply:
ok